src/EventSubscriber/TimestampableSubscriber.php

```diff
@@ -58,6 +64,25 @@ public function getSubscribedEvents(): array
         return [Events::loadClassMetadata];
     }
 
+    /**
+     * @return string[]
+     */
+    private function getTimestampableProperties(string $className): array
+    {
+        $properties = [];
+        foreach (['getCreatedAtProperties', 'getUpdatedAtProperties'] as $method) {
```

Comment:
Instead of magic, here can be `is_a()`

Reply:
@TomasVotruba modified this whendoing the change below.